go/pkg/indexerQL/indexerQL.go

```diff
@@ -0,0 +1,458 @@
+// Code generated by github.com/Khan/genqlient, DO NOT EDIT.
+
+package indexerql
```

Comment:
I would rename this package something more explicit like `gnoindexerql`

Reply:
Done :)